pulsar-client-api/src/main/java/org/apache/pulsar/client/api/AuthenticationFactory.java

```diff
@@ -61,6 +61,7 @@ public static Authentication token(Supplier<String> tokenSupplier) {
      *            the path to the TLS client private key
      * @return the Authentication object initialized with the TLS credentials
      */
+    @SuppressWarnings("checkstyle:MethodName")
```

Comment:
Use `CHECKSTYLE.OFF` and `CHECKSTYLE.ON` comments.

```
// CHECKSTYLE.OFF: MethodName
...
// CHECKSTYLE.ON: MethodName
```

Reply:
Thank you for the recommendation! What is the best way to use these comments?
```
// CHECKSTYLE.OFF: MethodName
public static Authentication TLS(String certFilePath, String keyFilePath) {
// CHECKSTYLE.ON: MethodName
// ... the rest of the method
}
```
or
```
// CHECKSTYLE.OFF: MethodName
public static Authentication TLS(String certFilePath, String keyFilePath) {
// ... the rest of the method
}
// CHECKSTYLE.ON: MethodName
```

Reply:
the later one?

Reply:
The code was updated

Reply:
The code was updated